rfcs/2022-05-17-11532-chronicle-sink.md

```diff
@@ -0,0 +1,239 @@
+# RFC 11532 - 2022-05-17 - Google Chronicle Sink
+
+This RFC proposes a new sink that sends data into
+[Google Chronicle](https://cloud.google.com/chronicle/docs).
+Chronicle is a Security Information and Event Managent (SIEM) platform to store,
+aggregate and search security telemetry.
+
+## Context
+
+- https://github.com/vectordotdev/vector/issues/11532
+
+## Cross cutting concerns
+
+- The schema support required for the UDM events relies on
+  [this PR](https://github.com/vectordotdev/vector/pull/11743)
+  being merged.
+
+## Scope
+
+### In scope
+
+- A sink that sends data into Google Chronicle. This includes UDM events and
+  unstructured log events.
```

Comment:
Why would UDM events be included? Isn't schema support required for this (cross cutting concerns) and SEIM event knowledge (out of scope)? And after looking at the [UDM schema](https://cloud.google.com/chronicle/docs/reference/udm-field-list) we can't populate 99% of these fields anyway, so what's the point?

Reply:
Got it, that makes sense.